package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Simple gulp plugin for the maven-deploy module",
   "main": "index.js",
   "scripts": {
-    "test": "./node_modules/.bin/mocha"
+    "test": "./node_modules/.bin/mocha && ./node_modules/.bin/finn-js-code-style index.js, sample/**"
```

Comment:
FYI `./node_modules/.bin` isnt neccessary here, as the npm client puts everything in `./node_modules/.bin` into `$PATH` when running these scripts